src/modules/ItineraryMakerModule/sections/Maps.tsx

```
@@ -1,23 +1,25 @@
   PlaceResult,
   Section,
 } from '../interface'
-import { customFetch } from '@/utils/newCustomFetch'
+import { customFetch } from '@/utils/customFetch'
 import { Globe, Phone } from 'lucide-react'
 import { Button } from '@/components/ui/button'
 import Link from 'next/link'
 import { decodePolyline } from '../utils'
+import {
+  AdvancedMarker,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)